.circleci/config.yml

```diff
@@ -1,9 +1,9 @@
-version: 2
+version: 2.1
 
 jobs:
     check-php-cs:
         docker:
-            - image: circleci/php:8.0-node-browsers
+            - image: cimg/php:7.1.33
```

Comment:
7.1 ?! Really?

Reply:
I was having another project in mind and I thought we were working on that one, monday mornings are tough sometimes.